api/groups/addressGroup.go

```diff
@@ -68,14 +68,9 @@ type esdtNFTTokenData struct {
 }
 
 // NewAddressGroup returns a new instance of addressGroup
-func NewAddressGroup(facadeHandler interface{}) (*addressGroup, error) {
-	if facadeHandler == nil {
-		return nil, errors.ErrNilFacadeHandler
-	}
-
-	facade, ok := facadeHandler.(addressFacadeHandler)
-	if !ok {
-		return nil, fmt.Errorf("%w for address group", errors.ErrFacadeWrongTypeAssertion)
+func NewAddressGroup(facade addressFacadeHandler) (*addressGroup, error) {
+	if facade == nil {
```

Comment:
check if nil for all tests like this one

Reply:
done